js/ai/src/model.ts

```diff
@@ -633,7 +634,8 @@ export interface ResolvedModel<
 
 export async function resolveModel<C extends z.ZodTypeAny = z.ZodTypeAny>(
   registry: Registry,
-  model: ModelArgument<C> | undefined
+  model: ModelArgument<C> | undefined,
+  options?: { warnDeprecated?: boolean }
```

Comment:
whence is it set?

Reply:
Oops! Refactored things one too many times and forgot to add the option. It's in `generate/action.ts` now.